test/scale-infra/README.md

```diff
@@ -0,0 +1,69 @@
+A scalable Kubernetes-based testbed for KubeStellar Performance Tests
+---------------------------------------------------------------------
+
+<img src="images/ks-scale-test-infra.png" width="60%" height="60%" title="ks-scale-test-infra">
+
+### Requirements
+AWS EC2 is the environment used to setup the testbed. So AWS credentials are required.
```

Comment:
Is it really only EC2? Aren't there networking and other AWS things involved too?

Reply:
You're correct! It's more than just creating EC2 - just fixed that sentence!